graspologic/align/orthogonal_procrustes.py

```diff
@@ -11,16 +11,16 @@ class OrthogonalProcrustes(BaseAlign):
     """
     Computes the matrix solution of the classical orthogonal Procrustes [1]_
     problem, which is that given two matrices `X` and `Y` of equal shape
-    (n, d), find an orthogonal matrix `self.Q_` that most closely maps `X` to
+    (n, d), find an orthogonal matrix ``Q_`` that most closely maps `X` to
     `Y`. Subsequently, uses that matrix to transform either the original `X`,
     or a different dataset in the same space.
 
     Note that when used to match two datasets, this method unlike
     :class:`~graspologic.align.SeedlessProcrustes`, not only requires that the
     datasets have the same number of entries, but also that there is some
     correspondence between the entries. In graph embeddings, this usually
-    corresponds to the assumption that the vertex i in graph X has the same
-    latent position as the vertex i in graph Y.
```

Comment:
it makes more sense to make things in the previous paragraph math if this is math, and vice versa, imo.

Reply:
I interpreted this second paragraph to be referring to the algorithm in general (hence math) rather than specific implementation.  If you wrote this and you intended it to refer to the specific input parameters ``X`` and ``Y`` then they should be double ticked.

Reply:
i was in referring to the specific implementation in these two paragraphs, and to general algo math in notes, but once again - whatever you think is more appropriate / consistent with the rest of the docs.